client_code/MultiSelectDropDown/__init__.py

```diff
@@ -228,6 +224,45 @@ def format_selected_text(self, count, total):
         return ", ".join(opt.title or opt.key for opt in self._options if opt.selected)
 
     ##### PROPERTIES #####
+    @property
+    def background(self):
+        return self._background
+
+    @background.setter
+    def background(self, value):
+        self._background = value
+        if value:
+            _S(_js.get_dom_node(self._select_btn)).children("button").css(
+                "background-color", value
+            )
+
+        pop(self._select_btn, "destroy")
+        popover(
+            self._select_btn,
+            self._dd,
+            placement="bottom-start",
+            arrow=False,
+            delay=0,
+            animation=False,
+            trigger="manual",
+            max_width="fit-content",
+            background=value,
+        )
```

Comment:
This feels a bit of a strange place to do popover creation.

Reply:
I fixed this by changing the popover so that the popover function returns the Popover object and adding background changing to the fui module's auto_update function